api/HTMLFormElement.json

```diff
@@ -214,10 +214,10 @@
               "version_added": null
             },
             "opera": {
-              "version_added": true
+              "version_added": "≤12.1"
```

Comment:
This is the `checkValidity` entry, not updated by https://github.com/mdn/browser-compat-data/pull/7056. Is that an oversight, or when was it added to Chrome?